app/controllers/split_checkout_controller.rb

```diff
@@ -16,10 +16,22 @@ class SplitCheckoutController < ::BaseController
   helper OrderHelper
 
   def edit
+    load_gateway_checkout_session
+    if @checkout_session&.session&.payment_status == "paid"
+      @checkout_session.payment.complete!
+      @checkout_session.save_card      
+      @order.confirm!
+      redirect_to order_path(@order) and return
+    end
     redirect_to_step unless params[:step]
   end
 
   def update
+    load_gateway_checkout_session
+    if ready_to_confirm? && redirect_url = @checkout_session&.session&.url
+      render js: "window.location='#{redirect_url}'" and return
```

Comment:
I don't think we should ever be rendering `js`. We should just be able to do `redirect_to redirect_url` here, no?

Reply:
I figured we had to do it this way since we can't use a `redirect_to` with a form that's using `data-remote`. When I used `redirect_to` the redirect showed up on the server logs but the page didn't change. There might well be a better way to do it though.